### PR TITLE
Issue #14 - Added Viewer Tools to ToolsCatalog and Added About Page

### PR DESCRIPTION
**Pull request by sesheppard**
Fixes Issue #14. New About page added to the branch, populated with the provided information about JSON, JSON LD, and Manifests. Adno and Universal Viewer added to ToolsCatalog file with the existing entries. Styles still need to be separated from the HTML documents, and further work likely still needs to be done on the layout and content of the About page.

### CODE REVIEW

File: web/about.html [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+            </div>
+            <p>Here, "@context": "https://www.com" tells computers to look up the meanings of "name" and "jobTitle" on www.com. This helps ensure that different systems interpret the data in the same way.</p>
+
+            <!--This description needs to be updated with additional information, possibly including a demo.-->
+            <h2>What is a JSON Manifest?</h2>
+            <p>A manifest provides information about a web application.</p>
+            <p>For more information on manifests, please visit:</p>
+            <p>https://iiif.io/api/cookbook/</p>
+
+            <!--This is not the correct way to prevent text from moving behind the page footer. I will need to figure out the issue and revise this shortly.-->
```
Comment (haneenalsewari) [inline web/about.html:218]: You can adjust the 'footer' CSS by increasing the bottom margin or applying a 'min-height' to the page to make sure that the content does not overlap

File: web/about.html [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+                <p>{</p>
+                <p>"@context": "https://www.com",</p>
+                <p>"name": "Jane Doe",</p>
+                <p>"jobTitle": "Software Developer"</p>
+                <p>}</p>
+            </div>
+            <p>Here, "@context": "https://www.com" tells computers to look up the meanings of "name" and "jobTitle" on www.com. This helps ensure that different systems interpret the data in the same way.</p>
+
+            <!--This description needs to be updated with additional information, possibly including a demo.-->
+            <h2>What is a JSON Manifest?</h2>
```
Comment (haneenalsewari) [inline web/about.html:213]: I agree with your comment! I think you need a more thorough explanation of JSON manifests, it's a little brief and doesn't have enough context for understanding.

**Comment by kate-holdener** [comment]
> New About page added to the branch, populated with the provided information about JSON, JSON LD, and Manifests. Adno and Universal Viewer added to ToolsCatalog file with the existing entries. Styles still need to be separated from the HTML documents, and further work likely still needs to be done on the layout and content of the About page.

Please use
Fixes #14
In the PR description

File: web/about.html [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+                background: #222222;
+            }
+
+            .header img {
+                display: block;
+                margin-left: auto;
+                margin-right: auto;
+                height: 80px;
+            }
+
```
Comment (alar12) [inline web/about.html:22]: Use separate external CSS file and also we should be using same fonts and their sizes across the website.

File: web/about.html [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+                <p>"jobTitle": "Software Developer"</p>
+                <p>}</p>
+            </div>
+            <p>Here, "@context": "https://www.com" tells computers to look up the meanings of "name" and "jobTitle" on www.com. This helps ensure that different systems interpret the data in the same way.</p>
+
+            <!--This description needs to be updated with additional information, possibly including a demo.-->
+            <h2>What is a JSON Manifest?</h2>
+            <p>A manifest provides information about a web application.</p>
+            <p>For more information on manifests, please visit:</p>
+            <p>https://iiif.io/api/cookbook/</p>
```
Comment (alar12) [inline web/about.html:216]: (https://iiif.io/api/cookbook/) links should be hyperlinked

**Review by alar12 — CHANGES_REQUESTED** [review]
Good Work!
Create new CSS file for styling and need more information on JSON Manifests like what they does and why they are created - visit: [IIIF Cookbook](https://iiif.io/api/cookbook/) and https://iiif.io/guides/using_iiif_resources/.

**Review by alar12 — APPROVED** [review]
Good Work!!, This works for the current sprint (need to modify CSS for footer)

**Pull request merged**